department-of-veterans-affairs/caseflow
# Ryan/appeals 29957

Resolves [Check Marks Not Needed in Receipt Date Filter Dropdown](https://jira.devops.va.gov/browse/APPEALS-29957)

# Description
These changes cause windows and mac clients to display the same dropdown, instead of the platform specific ones. These changes mean that the reciept date filter dropdown in reader will have a uniform look and feel across all platforms.

## Acceptance Criteria

## Testing Plan

1. Log into caseflow with a user who can access reader (BVADWISE works)
2. Navigate to Reader
3. Click the reciept date filter on a mac based computer and notice that the dropdown no longer displays the dark system color or a checkmark on the items.
4. Click the reciept date filter on a windows based computer and verify that the dropdown is the same as the mac version.

# Frontend
## User Facing Changes

 BEFORE|AFTER
<img width="202" alt="image" src="https://github.com/department-of-veterans-affairs/caseflow/assets/110805785/bb07bf99-a3a8-4d30-b6af-36c2908e8fb0">
|
<img width="198" alt="image" src="https://github.com/department-of-veterans-affairs/caseflow/assets/110805785/46987edb-044d-4b7f-978c-1e20bec81d5c">
